### DISCUSSION TITLE
Allow relations to/from schema collections

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#5819] [Feature Requests]
Currently, it's impossible to have a relationship to/from certain system collections like collections/fields/relations. This is due to the fact that the data within these endpoints doesn't come from table rows, but instead the database schema itself. This makes foreign key constraints impossible, as there aren't any keys to point to.

**Comment by averdesca** [comment]
could be possible to create a relation with a custom unique field (ex. autogenerated UUID field) created on one of the directus_* (fields/collection/relations) as foreign key?

**Reply by rijkvanzanten** [reply]
No unfortunately not. The rows in fields/collections/relations are optional meta data. Even when those rows don't exist, Directus will still work as expected based on the schema of the database itself
